js/index.js

```diff
@@ -1 +1,53 @@
-// Your code goes here
+// // Your code goes here
+
+let header = document.querySelector('.main-navigation');
+    //1
+    header.addEventListener('click', (event) => {
+        header.style.backgroundColor = 'cyan';
+    })
+    //2
+    header.addEventListener('dblclick', (event) => {
+        header.style.border = '5px solid pink';
+    })
+
+let btnDiv = document.querySelector('.content-pick');    
+    //3
+    btnDiv.addEventListener('mouseenter', (event) => {
+        btnDiv.style.backgroundColor = 'salmon';
+    })
+    //4
+    btnDiv.addEventListener('mouseleave', (event) => {
+        btnDiv.style.backgroundColor = 'lightyellow';
+    })
+
+let pyd = document.querySelector('.content-destination')
+    //5
+    pyd.addEventListener('mouseover', (event) => {
+        pyd.style.border = '5px dashed lightblue';
+    })
+
+    //6
+    .addEventListener('resize', (event) => {
+        
+    })
+
+let btns = document.querySelector('.btn');    
```

Comment:
`document.querySelector(".btn")` targets the first element with that class, not all the btns. You can also use document.querySelectorAll("class-name") to get all the elements with class name "class-name".
https://www.w3schools.com/jsref/met_element_queryselectorall.asp